controllers/suite_test.go

```diff
@@ -176,30 +173,35 @@ func createConfigMapSidecarConfig() {
 	nsObj.Name = "postgreslet-system"
 	Expect(svcClusterClient.Create(newCtx(), nsObj)).Should(Succeed())
 
-	bytes, err := os.ReadFile(filepath.Join(externalYAMLDirTest, "cm-sidecar.yaml"))
+	bytes, err := os.ReadFile(filepath.Join(externalYAMLDirTest, "configmap-sidecars.yaml"))
 	Expect(err).ToNot(HaveOccurred())
 
 	cm := &core.ConfigMap{}
 	Expect(yaml.Unmarshal(bytes, cm)).Should(Succeed())
+	// todo: Should not be hard-coded
```

Comment:
as discussed, please update the template and inject the correct values while generating the file in the `Makefile`